### DISCUSSION TITLE
plotting a list of functions

### DISCUSSION THREAD

**Discussion by atamulis** [sympy/sympy#25099] [Ideas]
This is a suggested addition to the documentation. I am new to Python and SymPy, but not at all new to computers and coding. I wanted to plot a family of functions, and it took me a lot of web searching to find the "argument unpacking operator" (https://stackoverflow.com/questions/16159365/sympy-plot-pass-in-list-of-symbolic-expressions) in order to find that I wanted:

plot( *[x/(1+x**2)**a for a in range(1,5)],(x,0,6))

The plot function's documentation is easy to find, and the web has many examples of Python's [<expression> for <variable> in <range>] notation, but finding that I needed a * before the first bracket was a struggle. An example like mine above in SymPy's Plotting module documentation would be very useful.

Thanks,

Andrius

**Comment by sylee957** [comment]
I don't think that sympy tutorial should contain everything about basic python tutorial though.
Because it means that everywhere in sympy should have tutorial for python which would be too much of unnecessity.
But if the example is beautiful for demonstrating capability of plot, it can be added.
